pennylane/wires.py

```diff
@@ -83,7 +94,7 @@ class Wires(Sequence):
     r"""
     A bookkeeping class for wires, which are ordered collections of unique objects.
 
-    If the input `wires` can be iterated over, it is interpreted as a sequence of wire labels that have to be
+    If the input ``wires`` can be iterated over, it is interpreted as a sequence of wire labels that have to be
```

Comment:
NIT: The class signature only says `Wires(Sequence)`, and does not name the input as `wires`.

Reply:
Isn't this making reference to the input of the constructor?

Reply:
I only added the second backtick since I thought this was a typo.

Reply:
I meant as in the class signature would have to be like
`Wires(wires: Sequence)` for you to be able to say input `wires`, otherwise people would be like, well technically input can have any name, if that makes sense.

Reply:
Oh yeah nvm